### PR TITLE
v2020.1.0 updates

### LINKED ISSUE #56: [BUG]: tgz files not removed on import bucket
When a bucket is imported, the tar file is extracted into `~/resen/bucket_name`.  This includes the manifest file and the mounted directories.  The mounted directories are extracted from `*.tgz` files, but these files are not removed after the mounting directory is extracted, such that the `~/resen/bucket_name` directory will contain something like both `data` (and extracted directory) and `data_mount.tgz` (the compressed directory that isn't actually doing anything).  To save space, these extra `*.tgz` files should be removed as part of the import process.

### PR DESCRIPTION
**Pull request by ljlamarche**
Updates in preparation for v2020.1.0 release.  This basically just revs version numbers, but it also fixes #56.

### CODE REVIEW

**Review by asreimer — COMMENTED** [review]
Looks good.

**Comment by asreimer** [comment]
I just added the 2020.1.0 image, which is available on dockerhub here: https://hub.docker.com/repository/registry-1.docker.io/earthcubeingeo/resen-core/tags?page=1

I then deleted all resen-core and resen-base images from my system.

Next, I updated my version of resen to use this PR, created a bucket, and everything worked as expected.

**Comment by asreimer** [comment]
Hmmm. Actually, set_sudo is failing. I'll see if I can fix that. 

When it fails, it fails to remove the '#' from this line in /etc/sudoers:

    #%sudo  ALL=(ALL:ALL) ALL

**Comment by asreimer** [comment]
There. The sudo bug (#60) is squashed.

**Comment by pmreyes2** [comment]
I have included changes to the documentation in a commit. I've proceeded as follows:

1. Check where 2019.1.0 in all the files
```
(v2020.1.0_updates)$ grep --include="*" -rnw . -e "2019.1.0"
./resen/Resen.py:838:        return [{"version":"2019.1.0","repo":"resen-core","org":"earthcubeingeo",
./docs/usage.rst:15:    Resen 2019.1.0 -- Reproducible Software Environment
./docs/usage.rst:57:     Available versions: 2019.1.0
./docs/usage.rst:58:     >>> Select a version: 2019.1.0
./docs/usage.rst:91:     Resen-core Version:  2019.1.0
./docs/usage.rst:113:    amber               2019.1.0                 running
./docs/usage.rst:125:    amber               2019.1.0                 exited
./docs/usage.rst:137:    amber               2019.1.0                 running
```

2. Change all instances in docs/usage.rst but the one of Available versions: 
Per:
[https://stackoverflow.com/questions/4247068/sed-command-with-i-option-failing-on-mac-but-works-on-linux](https://stackoverflow.com/questions/4247068/sed-command-with-i-option-failing-on-mac-but-works-on-linux)
This didn't work on my macbook:
$ sed -i 's/2019.1.0/2020.1.0/g' usage.rst
So something that work on all platforms (that have perl):
$ perl -i -pe's/2019.1.0/2020.1.0/g' usage.rst 

3. Edit docs/usage.rst Select a version to include all versions:
Available versions: 2019.1.0, 2020.1.0

4. ASSUMING the new tag will be 2020.1.0 edit docs/installation/installation.general.rst:
-pip install git+https://github.com/EarthCubeInGeo/resen.git@v2019.1.1
+pip install git+https://github.com/EarthCubeInGeo/resen.git@v2020.1.0

5. ASSUMING the new tag will be 2020.1.0 edit README.rst:
-3. Install Resen with pip ``pip install git+https://github.com/EarthCubeInGeo/resen.git@v2019.1.1``
+3. Install Resen with pip ``pip install git+https://github.com/EarthCubeInGeo/resen.git@v2020.1.0``

6. Add changelog

7. Adding information about sudo-enabled buckets including the password.

8. In readthedocs, I have compiled temporarily branch v2020.1.0_updates and the result can be seen here:
[https://resen.readthedocs.io/en/v2020.1.0_updates](https://resen.readthedocs.io/en/v2020.1.0_updates)

File: HISTORY.rst [hunk 1]
```diff
@@ -0,0 +1,23 @@
+
+.. :changelog:
+
+2020.1.0 (2020-06-23)
++++++++++++++++++++++
+
+- Add new available resen-core: 2020.1.0
+- bug fixed when importing a bucket: now tar files of mounted paths are removed after a successful import.
+- bug fixed when giving sudo privileges to locally run resen.
```
Comment (asreimer) [inline HISTORY.rst]:
```suggestion
- fixed bug causing enabling sudo access for user jovyan failed
```

**Review by asreimer — COMMENTED** [review]
One suggestion for change log.

File: HISTORY.rst (hunk 1 above)
Comment (pmreyes2) [reply HISTORY.rst]: now it looks better. BTW, there are other changes from 2019.1.0 to 2019.1.1 that I didn't include.

**Comment by pmreyes2** [comment]
I just changed the release day in the HISTORY.rst : 2020.1.0 (2020-06-24) hopping that we can release this today.

**Pull request merged**